doc/fwutil/fwutil.md

```diff
@@ -184,6 +201,8 @@ fwutil
 |    |--- version
 |    |--- status
 |    |--- updates -i|--image=<current|next>
+|    |--- updates -z|--fw-image=<fw_package.tar.gz>
```

Comment:
@sujinmkang what is the expected behaviour when --image=next and --fw-image=<fw_package.tar.gz> are provided simultaneously?

Reply:
We should take only one option. my current implementation takes the --fw-image option over --image option. Should we return error when user gives both option for one fwutil call?